playbooks/byo/openshift-cluster/config.yml

```diff
@@ -3,6 +3,19 @@
   tags:
   - always
 
+- name: Verify Requirements
+  hosts: OSEv3
```

Comment:
This will include other host types like [nfs] and [lb] which we don't want to apply our checks to. We should probably move this into playbooks/common/openshift-cluster/config.yml and use the oo_masters_to_config:oo_nodes_to_config:oo_etcd_to_config there.